sources/packages/web/src/components/students/StudentAssessmentDetails.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <request-assessment
+    :applicationId="applicationId"
+    :showWhenEmpty="false"
```

Comment:
mm....what is the use of this?`:showWhenEmpty="false"`

Reply:
As mentioned in the above comment, in student side of this story, the requests table needs to be hidden when the table is empty.